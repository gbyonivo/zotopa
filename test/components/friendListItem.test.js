import React from 'react';
import renderer from 'react-test-renderer';
import FriendListItem from '../../src/components/friendsListItem';

describe('FriendListItem', () => {
  it('should render correctly', () => {
    const actual = renderer.create(<FriendListItem friend={{ name: 'ivo', id: 1 }}/>);
    expect(actual).toMatchSnapshot();
  });
});