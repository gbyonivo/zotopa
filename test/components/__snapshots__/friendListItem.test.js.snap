// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FriendListItem should render correctly 1`] = `
<li
  className="sc-bdVaJa ijehez"
  onClick={[Function]}
>
  <span>
    ivo
  </span>
  <span />
</li>
`;
